zerver/tests/test_unread.py

```diff
@@ -138,12 +138,12 @@ def test_update_all_flags(self):
                                          Recipient.PERSONAL, "test2")]
 
         result = self.client_post("/json/messages/flags", {"messages": ujson.dumps(message_ids),
-                                                                 "op": "add",
+                                                           "op": "add",
```

Comment:
You should update the indentation of the following line in this and the next entry.

Otherwise this PR looks great!